packages/amplify_auth_cognito/example/lib/Widgets/ConfirmSignInWidget.dart

```diff
@@ -24,7 +24,8 @@ class _ConfirmSignInWidgetState extends State<ConfirmSignInWidget> {
     try {
       var res = await Amplify.Auth.confirmSignIn(
           confirmationValue: confirmationCodeController.text.trim());
-      widget.showResult('Confirm Sign In Status = ' + res.nextStep!.signInStep);
+      widget.showResult(
+          'Confirm Sign In Status = ${res.nextStep?.signInStep ?? 'null'}');
```

Comment:
Should be able to just do:
```dart
'Confirm Sign In Status = ${res.nextStep?.signInStep}');
```
If `res.nextStep?.signInStep` evaluates to `null` it'll get stringified as 'null'

Reply:
Nice thanks!